src/docs/Components/TextInput/index.mdx

```diff
@@ -13,6 +13,12 @@ status:
 
 <Preview nude path="input-state" />
 
+## Variations
+
+We have two different sizes of text inputs to answer differents needs. You can use both cases to suit your interface the best. Just keep in mind touch devices when placing smaller elements near each others.
+
+<Preview path="input-sizes" />
+
 ## Input with icon :
 
 Use a left icon to inform about a specific information type or formatting you expect from the user.
```

Comment:
I think it's better to structure like this :
## Variations (##)
### Sizes (###)
### Input with icon (###)

Reply:
fixed